src/gui/optionsdialog.h

```diff
@@ -181,7 +180,7 @@ private slots:
     SettingValue<QSize> m_storeDialogSize;
     SettingValue<QStringList> m_storeHSplitterSize;
 
-    QAbstractButton *m_applyButton;
+    QPushButton *m_applyButton;
```

Comment:
Apparently, you didn't learn any lesson from the previous PR... It's completely pointless to introduce this field.